docker/supervisor.d/supervisord.ini

```diff
@@ -0,0 +1,16 @@
+[supervisord]
+nodaemon=true
+
+[program:uwsgi]
+command=/usr/sbin/uwsgi --ini /etc/uwsgi/uwsgi.ini --die-on-term
+stdout_logfile=/dev/null
+stderr_logfile=/dev/null
```

Comment:
I'm no expert on this, but wouldn't you want:

```
stderr_logfile=/dev/stderr
stdout_logfile=/dev/stdout
```

What's your reasoning for not keeping the output?

Reply:
UWSGI server creates a lot of uneccesary noise which does not give any value, hence we tune that down